### PR TITLE
[Compute AG]GCP Hub-Target permissions

### PR DESCRIPTION
**Pull request by rcaballeromx**
We also need add the attachment to the cdn server. I included the path on the content already.

### CODE REVIEW

**Review by Pubs-MV — COMMENTED** [review]
you can remove this file from the book.

File: compute/admin_guide/agentless-scanning/onboard-accounts/onboard-gcp.adoc [hunk 1]
```diff
@@ -460,6 +460,12 @@ gcloud iam service-accounts keys create ${TARGET_PROJECT_NAME}-service_account_k
 [ "${GOOGLE_CLOUD_SHELL}X" == "trueX" ] && cloudshell download ${TARGET_PROJECT_NAME}-service_account_key.json
 ----
 
+. If you encounter the `failed to create snapshot for instance ...` error or `Error 403: Required 'compute.disks.createSnapshot' permission for ...` error, use this https://cdn.twistlock.com/docs/attachments/example-hub-target-permissions.tf.json[Terraform script] to grant the needed permissions across the target and hub accounts.
```
Comment (Pubs-MV) [inline compute/admin_guide/agentless-scanning/onboard-accounts/onboard-gcp.adoc]: @rcaballeromx the new location is 

https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/pdf/prisma/prisma-cloud/prerelease/example-hub-target-permissions.tf.json

Comment (rcaballeromx) [reply compute/admin_guide/agentless-scanning/onboard-accounts/onboard-gcp.adoc]: Done

**Comment by Pubs-MV** [comment]
Hey @rcaballeromx do we need to merge this or close this PR without merge?

**Comment by Pubs-MV** [comment]
closing this PR as the content is now causing merge conflicts. We can open a different PR in case we ned to host template

**Pull request closed without merging**